### DISCUSSION TITLE
Performance comparison between DNN vs Oqtane

### DISCUSSION THREAD

**Discussion by thienvc** [oqtane/oqtane.framework#1705] [Q&A]
Has anyone ever compared server and client performance between DNN and Oqtane?
Is there any official website using Oqtane production yet?
Sorry if I asked a silly question

**Comment by sbwalker** [comment]
@thienvc I am not aware of any performance comparisons between DNN and Oqtane but it would be great if someone was willing to volunteer. Obviously the performance test would depend on the scenario. Are you talking about comparing a default DNN installation vs. a default Oqtane installation and comparing page load for various default pages both unauthenticated and authenticated? 

Based on my own (non-scientific) experience of managing Oqtane sites in production, there is really no comparison - interacting with the administrative aspects of Oqtane is many orders of magnitudes faster than interacting with DNN. 

If you want to to see an Oqtane site which has been running in production for over a year now - you can look at:

https://dnfprojects.com/projects

It has a scheduled job which gathers data from Github everyday for .NET Foundation projects and visualizes the data both in summary and detailed forms.
